### DISCUSSION TITLE
feat(design): evaluate design and location of the landing page

### DISCUSSION THREAD

**Discussion by PatrickMi** [maevsi/maevsi#1358] [General]
Maevsi needs a landing page so we can present what maevsi is. This is good for users but also for event organizers who want to know who we are. Maevsi should show a cool landing page that presents maevsi and us, the people behind it.
It should also be evaluated where the landing page should be shown. Is it really maev.si where people who already know maevsi will access the page or should it be somewhere else like maev.si/about-us? 
Other sites like facebook.com and eventbrite.com also don't show who they are on their homepages.
CC @AlexSteckler @dargmuesli

**Comment by dargmuesli** [comment]
"about us" should already be covered by https://github.com/maevsi/maevsi/issues/294, is it?

**Comment by PatrickMi** [comment]
This issue is less about showing an about-us and more about evaluating what should be shown to the user when https://maev.si gets accessed.

**Comment by dargmuesli** [comment]
So this is meant more as a discussion than a task? Like find out what we want on a non-technical level first?

Then we could convert this issue into a discussion using the action on the right. Once the non-technical level is cleared, we could create an issue as a task with a clear TODO :raised_hands:

**Comment by dargmuesli** [comment]
There is now https://vibetype.de
